src/components/NumberInput/NumberInput.js

```diff
@@ -216,3 +181,49 @@ export default class NumberInput extends Component {
     );
   }
 }
+
+NumberInput.defaultProps = {
```

Comment:
Any reason why you moved `defaultProps`/`propTypes`?

Reply:
Reverted that change, for some reason I thought that was how the reset of the components were setup.